src/main/java/com/ffxivcensus/gatherer/GathererController.java

```diff
@@ -112,6 +116,10 @@ public boolean isConfigured() {
      * Method to gather data for characters in specified range.
      */
     private void gatherRange() {
+        // Firstly, clean the top-end of the database
+        LOG.debug("Cleaning top-end characters from the database");
+        playerRepository.trimDeleted(playerRepository.findLastNonDeleted());
```

Comment:
This searches the DB for the highest ID number that doesn't have the DELETED status, and removes all characters *after* it. Protects us from over-scraping and never picking new characters up in the over-scraped set.

May mean a few genuine newer deleted characters get removed from the DB prematurely, however they will get re-added the next cycle and will retain as long as there's a non-deleted character above them in the ID pool.